pandas/tests/scalar/timedelta/test_constructors.py

```diff
@@ -228,6 +228,13 @@ def test_overflow_on_construction():
         ("P0DT0H0M0.001S", Timedelta(milliseconds=1)),
         ("P0DT0H1M0S", Timedelta(minutes=1)),
         ("P1DT25H61M61S", Timedelta(days=1, hours=25, minutes=61, seconds=61)),
+        ("PT1S", Timedelta(seconds=1)),
+        ("PT0S", Timedelta(seconds=0)),
+        ("P1WT0S", Timedelta(days=7, seconds=0)),
+        ("P1D", Timedelta(days=1)),
```

Comment:
do we handle negatives of these? (IIRC that's on one of the issues)

Reply:
I didn't change the behavior of negativity, and the question is *how* to handle it.  One of the only references I can find regarding the expected behavior is [this comment](https://github.com/moment/moment/issues/2408#issuecomment-110180010), which also seems to indicate it's not very well defined in the ISO 8601 standard.  How would you feel about pulling that out into a separate issue?

Reply:
yes pls create a separate issue. only thing for now is do we raise if a negative is passed in? (i think we should unless / until we decide on how to handle it), though isn't the result just negative of the Timedelta itself which is well defined.

Reply:
Ok, will do.  

No, nothing is raised when it is negative, and the behavior is, admittedly, quite odd, i.e.

`"P-6DT0H50M3.010010012S"` parses as `Timedelta(
                days=-6,
                minutes=50,
                seconds=3,
                milliseconds=10,
                microseconds=10,
                nanoseconds=12,
            )`
, and the negative is only allowed right after the P descriptor.  A negative in any other position will raise an error.  As far as I can tell, there are two possibilities to go here:

- only support an overall "-" e.g. `"-P6DT1H" = Timedelta('-7 days +23:00:00')` and/or
- support positive/negative on each, e.g. `"P7DT-1H3M" = Timedelta('6 days 23:03:00')`

Reply:
Issue now here: #37172.  I would prefer to change/update the negativity behavior in a separate PR if that's ok.

Reply:
yep that's great. well scoped PRs are good. one thing at a time. and thanks for working on this.